### ISSUE TITLE
How to apply census transform to images in Data preprocess(in DDFlow paper4.1)

### ISSUE THREAD

**Issue by daodan007** [ppliuboy/DDFlow#15]
I know census loss and had found the implement in your code, but I haven't found the census transform in data preprocess procedure in the code.
I found the raw image was sent to feature extractor directly in eval procedure, while only been done some flip/channel swap in training. I don't know where and how the census transform preprocessing is conducted?
I also refer the Unflow's paper, ternary census transform is implemented in the loss function(not in preprocess). Trying to read "Why is the Census Transform Good for Robust Optic Flow Computation?", but is too difficute to understand.
Could you help me to figure it out?

**Comment by ppliuboy** [comment]
For the input to the neural network, we do not employ census transform. Census transform is only employed when computing photometric difference.
